### PR TITLE
SUIT-9649 Errors from `getComposers` are not reported to Sentry

### PR DESCRIPTION
**Pull request by AntonSBilan**
SUIT-9649
Errors from `getComposers` are not reported to Sentry

### CODE REVIEW

**Comment by andriichumak** [comment]
@AntonSBilan , can't you use https://mineus.atlassian.net/browse/SUIT-9581 ?

**Comment by AntonSBilan** [comment]
@andriichumak Tried `Raven.wrap` but it does not report error, not sure why, but at least `shouldSendCallback` not fired

File: lib/utils/sentry/Raven.js [hunk 1]
```diff
@@ -37,6 +37,20 @@ function captureException(error) {
 	return new Promise(res => Raven.captureException(error, res));
 }
 
+function wrap(func) {
```
Comment (andriichumak) [inline lib/utils/sentry/Raven.js]: Please, reformat function according to aslant rules

**Comment by andriichumak** [comment]
@AntonSBilan Are you sure your version of works as expected? I was able to get one Sentry error, but then I changed something in code, removed error from Sentry logs and ran it again - was unable to receive any more reports. It seems that Setry's `captureException` is async and sometimes process ends before it's sent. Could you double-check it, please.

File: lib/utils/makeChain.js [hunk 2]
```diff
@@ -1,6 +1,7 @@
 const {mergeAll} = require('ramda');
+const Raven = require('raven');
 
-const makeChain = (getComposers, data) => {
+const makeChain = Raven.wrap((getComposers, data) => {
```
Comment (givehug) [inline lib/utils/makeChain.js]: It should work, I was able to send reports constantly. I think it matters where you put Raven.wrap( statement.

Comment (AntonSBilan) [reply lib/utils/makeChain.js]: that's wired,  I tried it on my mac and it does not even get to 'shouldSendCallback' method (Raven creates new domain and run wrapped function and for me domain.on(error) not fired)

**Pull request merged**